entoas/annotation.go

```diff
@@ -146,14 +147,14 @@ func (a Annotation) Merge(o schema.Annotation) schema.Annotation {
 	return a
 }
 
-func (op OperationConfig) merge(other OperationConfig) OperationConfig {
+func (op *OperationConfig) merge(other OperationConfig) OperationConfig {
```

Comment:
Let's remove the return value completely then, we are not using it anywhere anyways ;-)